src/components/page/sectionBreak.jsx

```diff
@@ -7,7 +7,14 @@ import Block from './blocks/index';
 class SectionBreak extends React.PureComponent {
   render() {
     const { blocksGroups, shared: blockShared, t } = this.props;
-    const sectionBreakImagePath = '/images/section_break_celebration.gif';
+    const { section } = blockShared;
+    const { sections } = this.global;
+    const isFinalBreak = section === sections.length - 2;
```

Comment:
since `sectionOrder` is used to create the sections in the global store, it can be used to re-access which section this break is for and in this case we want to see if it is in the next to last section, since the final section is ended by the summary page and QA review